golang-github-osbuild-composer.spec

```diff
@@ -40,7 +40,7 @@ BuildRequires:  golang(github.com/stretchr/testify)
 
 Requires: golang-github-osbuild-composer-worker
 Requires: systemd
-Requires: osbuild >= 12
```

Comment:
This should probably be 15-{2,3} to pick up the patch that @gicmo added. But we could do that in a follow-up.